src/sections/aliases.tsx

```diff
@@ -0,0 +1,211 @@
+      <h2>Aliases table</h2>
+      <button onClick={() => {
+        if (aliases) {
+          exportToCsv(Object.keys(aliases).map(alias => ({
+            alias: alias,
+            peerId: aliases[alias]
+          })), "aliases.csv")
+        }
+      }}>export</button>
+      <CSVUploader onParse={(parsedData) => {
+        for (const data of parsedData) {
+          if (data.alias && data.peerId && loginData.ip && loginData.apiKey) {
+            dispatch(actionsAsync.setAliasThunk({
+              alias: String(data.alias),
+              peerId: String(data.peerId),
+              apiEndpoint: loginData.ip,
+              apiToken: loginData.apiKey
+            }))
+          }
+        }
+      }} />
+      <TableContainer component={Paper}>
+        <Table sx={{ minWidth: 650 }} aria-label="aliases table">
+          <TableHead>
+            <TableRow>
+              <TableCell>id</TableCell>
+              <TableCell>peerId</TableCell>
+              <TableCell>alias</TableCell>
+              <TableCell>actions</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {Object.entries(aliases ?? {}).map(([alias, peerId], key) => (
+              <TableRow
+                key={key}
+              >
+                <TableCell component="th" scope="row">
+                  {key}
+                </TableCell>
+                <TableCell>{peerId}</TableCell>
+                <TableCell>{alias}</TableCell>
+                <TableCell><DeleteAliasButton alias={alias} /></TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </Section>
+  );
+}
+
+function DeleteAliasButton({ alias }: { alias: string }) {
+  const dispatch = useAppDispatch();
+  const loginData = useAppSelector(selector => selector.auth.loginData)
+
+  return <button onClick={() => {
+    if (loginData.ip && loginData.apiKey) {
+      dispatch(actionsAsync.removeAliasThunk({ alias, apiEndpoint: loginData.ip, apiToken: loginData.apiKey }))
+    }
+  }}>delete</button>
+}
+
+function CreateAliasForm() {
+  const dispatch = useAppDispatch();
+  const loginData = useAppSelector(selector => selector.auth.loginData)
+
+  const [form, setForm] = useState<{ peerId: string; alias: string }>({
+    alias: '',
+    peerId: ''
+  })
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target
+    setForm({ ...form, [name]: value })
+  }
+
+  return (
+    <div>
+      <input type="text" name="peerId" placeholder="peerId" onChange={handleChange} value={form.peerId} />
+      <input type="text" name="alias" placeholder="alias" onChange={handleChange} value={form.alias} />
+      <button onClick={() => {
+        if (loginData.ip && loginData.apiKey) {
+          dispatch(actionsAsync.setAliasThunk({ alias: form.alias, peerId: form.peerId, apiEndpoint: loginData.ip, apiToken: loginData.apiKey }))
```

Comment:
If that call will fail, we need an error to show up under the form with information what went wrong.
If the call will succeed, we show information under the form that it;'s saved and we clear the textfield.